### PR TITLE
레지스터 페이지 작업, 2차 -2 (StickyBottomButton)

### PR DESCRIPTION
**Pull request by chepchep2**
2차 - 2 작업입니다.

### CODE REVIEW

**Comment by chepchep2** [comment]
* **#17** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/17?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#16** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/16?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#15** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/15?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#14** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/14?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#13** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/13?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#12** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/12?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#11** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/11?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#10** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/10?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#9** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/9?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#8** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/8?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/7?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#6** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/6?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#5** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/5?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/4?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#3** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/3?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/2?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/introPage`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @chepchep2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/common/sticky_bottom_button.dart [hunk 1]
```diff
@@ -19,7 +19,7 @@ class _StickyBottomButtonState extends State<StickyBottomButton> {
   @override
   Widget build(BuildContext context) {
     return BottomAppBar(
-      color: Colors.grey,
+      color: Colors.grey[300],
```
Comment (flowkater) [inline lib/common/sticky_bottom_button.dart:22]: Color 는 const 상수로 빼서 쓰면 좋을 것 같아요.

**Review by flowkater — COMMENTED** [review]
각 브랜치로 다시 돌아가서 수정하고 업데이트하는게 맞을 것 같은데, 저도 방법을 제대로 테스트를 못해봐서 일단 참고만하세요

File: lib/common/sticky_bottom_button.dart (hunk 1 above)
Comment (chepchep2) [reply lib/common/sticky_bottom_button.dart:22]: 알겠습니다. 상수로 빼는 걸로 수정하겠습니다

**Comment by chepchep2** [comment]
> 각 브랜치로 다시 돌아가서 수정하고 업데이트하는게 맞을 것 같은데, 저도 방법을 제대로 테스트를 못해봐서 일단 참고만하세요
> 
> 
넵

**Comment by flowkater** [comment]
### Merge activity

* **Mar 7, 8:27 AM EST**: @flowkater started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/5).
* **Mar 7, 8:36 AM EST**: [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/5) rebased this pull request as part of a merge.
* **Mar 7, 8:37 AM EST**: @flowkater merged this pull request with [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/5).

**Review by flowkater — APPROVED** [review]

**Pull request merged**